string station_name
string robot_address
geometry_msgs/Point start
bool visit_all_fields
unomas/FieldPlan[] fields
unomas/RoutePlan[] routes
